kinetic/distribution.yaml

```diff
@@ -1546,6 +1546,25 @@ repositories:
       url: https://github.com/OTL/cv_camera.git
       version: master
     status: developed
+  darknet_ros:
+    doc:
+      type: git
+      url: https://github.com/leggedrobotics/darknet_ros.git
+      version: master
+    release:
+      packages:
+      - darknet_ros
+      - darknet_ros_msgs
+      tags:
+        release: release/kinetic/{package}/{version}
+      url: https://github.com/leggedrobotics/darknet_ros-release.git
+      version: 1.1.0-0
+    source:
+      test_pull_requests: true
```

Comment:
Have you followed the [instructions](http://wiki.ros.org/buildfarm/Pull%20request%20testing) to provide access to this repo or the org unit to the @ros-pull-request-builder user? If not, either remove this line or provide the necessary access as described in the wiki page.

Reply:
Thanks for the help! I will do the necessary steps and let you know as soon as I finished.